src/wrapper/jnienv.rs

```diff
@@ -983,6 +983,9 @@ impl<'a> JNIEnv<'a> {
     ///
     /// This entails a call to `GetStringUTFChars` and only decodes java's
     /// modified UTF-8 format on conversion to a rust-compatible string.
+    ///
+    /// # Panics
+    /// This call panics when given an Object that is not a java.lang.String
```

Comment:
Nitpick: let's add a new line between the "panics" and the explanation as it is done in [std](https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#438).